### PR TITLE
:bug: FIX preloader loading everytime

### PR DESCRIPTION
**Pull request by kud04rk**
added cookie to check if the user visited the website first time.
added three functions:
setCookie
getCookie
checkCookie

### CODE REVIEW

File: js/preloader.js [hunk 1]
```diff
@@ -1,147 +1,178 @@
...
+  $(document).scrollTop($(document).height()); 
+}
+
+function setCookie(cname,cvalue,exdays) {
+  var d = new Date();
+  d.setTime(d.getTime() + (exdays*24*60*60*1000));
+  var expires = "expires=" + d.toGMTString();
+  document.cookie = cname + "=" + cvalue + ";" + expires + ";path=/";
+}
+function getCookie(cname) {
```
Comment (saglaniharsh) [inline js/preloader.js]: Please change the getCookie method to something more shorter. Probably something like [this](https://stackoverflow.com/a/25490531/6303750)
Also, do not forget to mention this link in the comment if you use this code.

File: js/preloader.js [hunk 2]
```diff
@@ -1,147 +1,178 @@
...
+      c = c.substring(1);
+    }
+    if (c.indexOf(name) == 0) {
+      return c.substring(name.length, c.length);
+    }
+  }
+  return "";
+}
+function checkCookie() {
+  var user=getCookie("LAKSHYA#CTF#INC#2020");
```
Comment (saglaniharsh) [inline js/preloader.js]: Change the name of cookie to "visited"
and value to 1
instead of "LAKSHYA#CTF#INC#2020"

File: js/preloader.js [hunk 3]
```diff
@@ -1,147 +1,178 @@
...
+  if (user != "") {    
+    createLog("ok");
```
Comment (saglaniharsh) [inline js/preloader.js]: Also, can you mention your thoughts on why you're calling createLog?

**Review by saglaniharsh — CHANGES_REQUESTED** [review]

File: js/preloader.js (hunk 1 above)
Comment (kud04rk) [reply js/preloader.js]: @ashawe thanks for the feedback and the reference I have made changes and mentioned the comment as well . please review

File: js/preloader.js (hunk 2 above)
Comment (kud04rk) [reply js/preloader.js]: @ashawe thanks for the feedback and I have made the changes you have requested(Change the name of cookie to "visited"
and value to 1 )

File: js/preloader.js (hunk 3 above)
Comment (kud04rk) [reply js/preloader.js]: @ashawe It was a mistake i left it while debugging. I have made changes and removed the createLog.
Please review the Changes.

**Comment by kud04rk** [comment]
@ashawe I have made the changes that were requested.

**Review by saglaniharsh — APPROVED** [review]

**Pull request merged**